ConsoleApp1/sbas_homework_reflection.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace ConsoleApp1
+{
+    internal class sbas_homework_reflection
+    {
+        public void Foo()
+        {
+            string path = Path.Combine("D:\\C#\\Test_snake\\bin\\Debug\\net6.0", "Test_snake.dll"); 
+            var assembly = Assembly.LoadFrom(path);
+
+            Type typeAssembly = assembly.GetType();
+
+            foreach (var item in assembly.GetTypes())
+            {
+                Console.WriteLine("________________Class_____________\n");
+
+                Console.WriteLine($"Name of namespace: {item.Namespace}, Name of types: {item.Name}, Base type: {item.BaseType}");
+                Console.WriteLine("__________________Property_______________\n");
+                foreach (var item1 in item.GetProperties())
```

Comment:
Does it display private, static, protected properties? Same question for methods